.github/workflows/data/robot_ws/src/m2wr_description/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
Also, when fetching our sample apps, fetch it to the [default destination directory](https://github.com/aws-robotics/aws-robomaker-sample-application-helloworld/blob/1d8342a8dd4bc74f8a3f17b3192aec4bf1357cb9/.github/workflows/ros1.yml#L39), so that this repo's test workflow YAML will resemble more closely to that of a user's.

Reply:
When I tried this, action checkout will fails if we checkout another git repos. The clean solution for multiple repos is to checkout them into different folders.

Reply:
You can specify the path where you fetch/checkout to: https://github.com/actions/checkout#checkout-multiple-repos-side-by-side

I ask that you fetch/checkout the sample apps to the default location, which means the GitHub action will need to be checked out to a non-default location. (The repo that's being checked out to the default location may need to be the first one that's checked out, ie. the sample app will need to be checked out before the GitHub action.)